### DISCUSSION TITLE
Any plans to migrate this action to Node 20?

### DISCUSSION THREAD

**Discussion by ntroliveira** [mathze/workflow-dispatch-action#113] [Q&A] [answered]
https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/

Thank you

**Comment by mathze** [comment, marked as answer]
Hi @ntroliveira,

thank you for point me to the upcoming change.
And yes, I try will update to Node 20.

**Reply by ntroliveira** [reply]
Thank you very much @mathze
